Frontend/keyGenerator.py

```
@@ -0,0 +1,125 @@
+import os
+import re
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)